template.yaml

```diff
@@ -200,6 +200,19 @@ Resources:
               - s3:GetObject
             Resource: !Sub ${StructuredFieldsBucket.Arn}/*
 
+  DisableCloudWatchLoggingPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Deny
```

Comment:
This `Deny` statement [overrides](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_evaluation-logic.html#AccessPolicyLanguage_Interplay) an allow statement which is attached to this lambda (I can't see this policy in this CloudFormation file; presumably it's being added under the hood by the SAM):

![image](https://user-images.githubusercontent.com/19384074/140069666-0ebfcb47-9516-468d-aac7-02557216edc9.png)